brambling/static/brambling/sass/modules/_list-item.sass

```diff
@@ -0,0 +1,5 @@
+.list-item
+	display: flex
+	flex-direction: row
+	justify-content: space-between
+	align-items: center
```

Comment:
After some deliberation in Slack, we have determined that no one cares about IE9 (only 0.18% of traffic for the past 365 days) and everything else more or less supports flexbox. (Though we should double-check that we have a working autoprefixer setup?)